stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -192,32 +193,43 @@ enum AttributeCode {
   ///   }
   kDenseIntOrFPElementsAttr = 21,
 
+  ///   ArrayAttr {
+  ///   DictionaryAttr {
+  ///     attrs: <StringAttr, Attribute>[]
+  ///   }
+  kDictionaryAttr = 22,
```

Comment:
Reusing this constant in this specific case LGTM since we don't provide stability guarantees for VHLO bytecode yet. Same below.

Reply:
That's the thought. After we roll out our first version, these numbers will not be allowed to change.